2996_reflection/reflection.md

```diff
@@ -4535,9 +4570,9 @@ consteval info value_of(info r);
 consteval info parent_of(info r);
 ```
 
-[#]{.pnum} *Constant When*: `r` represents a member of either a class or a namespace.
+[#]{.pnum} *Constant When*: `r` represents a variable, structured binding, function, enumerator, class, class member, bit-field, template, namespace, alias of a type or namespace, or base specifier.
 
-[#]{.pnum} *Returns*: A reflection of that entity's immediately enclosing class or namespace.
+[#]{.pnum} *Returns*: A reflection of that entity's immediately enclosing class, function, or namespace.
```

Comment:
Did we want to have reflections of scopes? For instance:
```cpp
void f() {
    int x;
    if (true) {
        int y;
    }
}
```
I don't think we want `parent_of(^x) == parent_of(^y)`?

Reply:
I think we do. I'm not clear that Clang has a natural (persistent) representation of the scope containing `y`; scopes are discarded as they're closed. My understanding is that Clang tends to retain more information than many implementations, so I think it may not be unique in having that problem. We don't have any implementation experience representing block scopes; I don't think we should introduce reflections of them "by fiat".